AstarTileMap.gd

```diff
@@ -45,13 +45,13 @@ func add_unit(unit: Object) -> void:
 func remove_unit(unit: Object) -> void:
 	units.erase(unit)
 
-func position_has_obstacle(obstacle_position: Vector2, ignore_obstacle_position := null) -> bool:
+func position_has_obstacle(obstacle_position: Vector2, ignore_obstacle_position = null) -> bool:
```

Comment:
Oh nice catch! I think some type is better than nothing. In this case it might make sense to use `Vector2` and infinite like so:

```gdscript
func position_has_obstacle(obstacle_position: Vector2, ignore_obstacle_position := Vector2.INF) -> bool:
```